odk1-src/briefcase-using.rst

```diff
@@ -204,9 +217,25 @@ To get help about the command line operation:
 Pulling form data from Aggregate
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: console
+- CLI flag: `-plla` or `--pull_aggregate`
```

Comment:
I think there is a better (native to Sphinx or in a plugin) to document command line options,
similar to the way we document functions.

I will try to look into this.